common/supplemental/supplementalData.xml

```diff
@@ -1544,7 +1540,7 @@ XXX Code for transations where no currency is involved
 		<language type="fa" territories="PK" alt="secondary"/>
 		<language type="fan" scripts="Latn"/>
 		<language type="fan" territories="GQ" alt="secondary"/>
-		<language type="ff" scripts="Latn Adlm"/>
+		<language type="ff" scripts="Adlm Latn"/>
```

Comment:
Hmm. The "ff" files currently in common/main have Latn as the default script. We have some ff_Adlm files in seed. I changing the default sript here going to cause problems with the current data configuration?

Reply:
My understand is that "primary" script doesn't mean "default" script, and that order is not relevant here ( i.e. the tooling puts them in alphabetical order ).  I was looking at Serbian to notice that it has two "primary" scripts.